src/services/cache-config/CacheConfigService.ts

```diff
@@ -345,6 +344,25 @@ export class CacheConfigService {
     }
   }
 
+  private transactionHelper(builder: CompositeOpBuilder, address: string[], config: any, create: boolean): void {
+    let validTxMode: boolean = true;
+    if (isNotNullOrUndefined(config.transaction) && isNotNullOrUndefined(config.transaction.TRANSACTION)) {
```

Comment:
If `config.transaction` or `config.transaction.TRANSACTION` are null, should we not exist this method and do nothing? Currently if they are null/undefined, we will always go into either the create/update helper